doc/classes/TileMap.xml

```diff
@@ -188,6 +188,7 @@
 			<argument index="0" name="arg0" type="Vector2">
 			</argument>
 			<description>
+				Apply autotiling rules for the cell referenced by its grid-based X and Y coordinates, and its surrounding cells.
```

Comment:
- Use present tense in all method descriptions: Apply -> Applies
- Remove unnecessary comma